app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/ChatViewHolder.kt

```diff
@@ -49,6 +50,7 @@ class ChatViewHolder(view: View, clickListener: MessageViewHolder.ClickListener,
     fun setView(model: ChatMessage?, viewType: Int, context: Context) {
         if (model != null) {
             this.model = model
+            contexts = context
```

Comment:
Why?

Reply:
I tried to get the context via requireContext in setRatings, but I didn't get it. Also getContext, this failed. Hence, I came up with this id of storing the context in a global variable and using it in this function.

Reply:
No need to store in global variable, just pass it as an argument to the function

Reply:
> Why?

Okk sir